eclair-core/src/main/scala/fr/acinq/eclair/db/sqlite/SqliteNetworkDb.scala

```diff
@@ -155,11 +163,13 @@ class SqliteNetworkDb(sqlite: Connection) extends NetworkDb {
           chainHash = null,
           shortChannelId = ShortChannelId(rs.getLong("short_channel_id")),
           timestamp = rs.getLong("timestamp"),
-          flags = rs.getBytes("flags"),
+          messageFlags = rs.getBytes("flags")(0),
+          channelFlags = rs.getBytes("flags")(1),
           cltvExpiryDelta = rs.getInt("cltv_expiry_delta"),
           htlcMinimumMsat = rs.getLong("htlc_minimum_msat"),
           feeBaseMsat = rs.getLong("fee_base_msat"),
-          feeProportionalMillionths = rs.getLong("fee_proportional_millionths"))
+          feeProportionalMillionths = rs.getLong("fee_proportional_millionths"),
+          htlcMaximumMsat = Option(rs.getLong("htlc_maximum_msat")))
```

Comment:
The problem is that rs.getLong will return 0 is the column contains a NULL value (because it returns a `long` that cannot be NULL and not a `Long` object), we should then call `rs.wasNull()`.

For example we could use:

```scala
      def getNullableLong(label: String) : Option[Long] = {
        val result = rs.getLong(label)
        if (rs.wasNull()) None else Some(result)
      }
```